CONTRIBUTING.md

```diff
@@ -321,17 +321,24 @@ npm run release-maintenance
 
 **Be sure to use `$ npm run` instead of `$ yarn run` to avoid issues**
 
-#### Beta version
+#### Beta version β
 
-Beta version release is available on any branch except `master`, `maintainance` or `develop`. The
+Beta version release is available on any branch except `master`, `maintainance`. The
```

Comment:
I don't understand this sentence

Reply:
Can you tell what you understand from this sentence?

Reply:
What the difference between the branches is, why develop needs the flag

Reply:
The difference is that the default behaviour with develop is to do a complete release.